game/scripts/vscripts/units/twin.lua

```diff
@@ -0,0 +1,29 @@
+LinkLuaModifier( "modifier_boss_phase_controller", "modifiers/modifier_boss_phase_controller", LUA_MODIFIER_MOTION_NONE )
+
+function Spawn (entityKeyValues)
+  thisEntity:FindAbilityByName("boss_twin_twin_empathy")
+  thisEntity:FindAbilityByName("boss_twin_spawn_twin")
+
+  thisEntity:SetContextThink( "TwinThink", partial(TwinThink, thisEntity) , 1)
+  print("Starting AI for " .. thisEntity:GetUnitName() .. " " .. thisEntity:GetEntityIndex())
+
+  ABILITY_spawn_twin = thisEntity:FindAbilityByName("boss_twin_spawn_twin")
+  ABILITY_empathy = thisEntity:FindAbilityByName("boss_twin_twin_empathy")
+
+  local phaseController = thisEntity:AddNewModifier(thisEntity, ABILITY_empathy, "modifier_boss_phase_controller", {})
+  phaseController:SetPhases({ 75, 50 })
+  phaseController:SetAbilities({
+    "boss_twin_twin_empathy"
+  })
+end
+
+function TwinThink()
```

Comment:
Rename this function to something indicative of it's purpose, such as `SpawnDumbTwin`. Don't forget to change it above, as well.